### ISSUE TITLE
Excessively long payout address causes Timeout waiting for NTP

### ISSUE THREAD

**Issue by computeronix** [Storj/storjshare-gui#536]
### Versions

Replace the values below with your own:

- **GUI Version:** v4.1.1
- **Operating System:** Windows 10 Enterprise 1607

### Expected Behavior

Please describe the program's expected behavior.

```
When you put in a payout address (regardless of size) it should say "Invalid payout" if its invalid.

```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
When you put in a payout address excessively long it hangs, then eventually says invalid payout and then generates the below screenshot


```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1.Enter a payout address of

`a` repeated 82945 times

### Screenshots (Optional)

If the error is graphical in nature it is helpful to provide a screenshot. 

![error](https://cloud.githubusercontent.com/assets/19168174/25106117/e3791b4a-2395-11e7-911a-21444a541181.PNG)

**Comment by computeronix** [comment]
As part of the above you may want to limit the text size of the textbox to prevent anything over xx from being entered

**Comment by braydonf** [comment]
I truncated the repeated `a` in steps to reproduce so this issue can be read.

**Comment by computeronix** [comment]
thanks!  also I tried the same in directory and it instantly failed with invalid directory where as the more content you put in the SJCX address the program seems to hang more and more then fail

**Comment by bookchin** [comment]
Okay, but why would you ever do this and why should we account for intentional borking of a config file or someone's cat taking a nap on their keyboard?

I imagine the reason it hangs is that it is intact trying to validate the address.

**Comment by computeronix** [comment]
Good point but the other way to look at it, is why would it hang regardless of whats put in the box and then display something different than expected.

As far as data validation should be checked within constraints just to prevent anything in the future from creeping up.

**Comment by JukeboxRhino** [comment]
Fixed with #623
